Update active-directory-b2c-setup-aad-custom.md

marcelodiiorio opened the pull request:
We have updated the sample code above, but we forgot to update the step by step procedure below.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

marcelodiiorio committed:
Update active-directory-b2c-setup-aad-custom.md

We have updated the sample code above, but we forgot to update the step by step procedure below.
